projects/hslayers/package.json

```diff
@@ -56,6 +56,9 @@
     "xml-js": "^1.6.11"
   },
   "dependencies": {
-    "tslib": "^2.0.0"
+    "@angular/platform-browser": "^12.1.0",
```

Comment:
@fzadrazil I suppose these deps were not intended.

Reply:
Sure not, I have also noticed and will push a fix in a minute.